crates/forge/src/scarb/config.rs

```diff
@@ -0,0 +1,123 @@
+use anyhow::{anyhow, bail, Result};
+use conversions::StarknetConversions;
+use itertools::Itertools;
+use serde::Deserialize;
+use starknet::core::types::{BlockId, BlockTag};
+use std::collections::HashMap;
+use url::Url;
+
+#[derive(Debug, PartialEq, Default)]
+#[allow(clippy::module_name_repetitions)]
+pub struct ForgeConfig {
+    /// Should runner exit after first failed test
+    pub exit_first: bool,
+    /// How many runs should fuzzer execute
+    pub fuzzer_runs: Option<u32>,
+    /// Seed to be used by fuzzer
+    pub fuzzer_seed: Option<u64>,
+
+    pub fork: Vec<ForkTarget>,
+}
+
+#[derive(Debug, PartialEq, Clone)]
+pub struct ForkTarget {
+    pub name: String,
+    pub url: Url,
```

Comment:
```suggestion
    pub url: String,
```
we should keep validation in one place

Reply:
If we don't parse it when parsing a config it won't be obvious to the user that the unparsable url is in Scarb. We don't do that in compilation (since we want to split compilation and url parsing) but I think if we can provide better UX without a lot of cost in terms of code abstraction we should do that

Reply:
Nvm, see below